src/core/dbus-manager.c

```diff
@@ -2233,8 +2233,7 @@ static int install_error(
                 int c,
                 InstallChange *changes,
                 size_t n_changes) {
-
```

Comment:
We usually insert an empty line between function declaration and variable declaration. So, please keep the empty line.

Reply:
Done.